http/events.go

```diff
@@ -19,25 +20,34 @@ import (
 	"github.com/hashicorp/vault/sdk/logical"
 	"github.com/hashicorp/vault/vault"
 	"github.com/hashicorp/vault/vault/eventbus"
+	"github.com/patrickmn/go-cache"
+	"github.com/ryanuber/go-glob"
 	"nhooyr.io/websocket"
 )
 
-type eventSubscribeArgs struct {
+// DefaultWebSocketRevalidationTime is how often we re-check access to the
+// events that the websocket requested access to.
+const DefaultWebSocketRevalidationTime = 5 * time.Minute
```

Comment:
This has the name `Default`, but I don't see a way to configure it as part of this PR. If there isn't going to be a way to configure it, I'd prefer we nix the word `Default`, as it suggests to me that there will be a way to configure it.

Reply:
I was debating adding a way to set a configuration option for this. But I can rename this for now, and we can always make this configurable later.